ddtrace/contrib/psycopg/async_cursor.py

```diff
@@ -21,9 +21,6 @@ async def __aexit__(self, exc_type, exc_val, exc_tb) -> None:
         # messages will be the same.
         await self.__wrapped__.__aexit__(exc_type, exc_val, exc_tb)
```

Comment:
Nit: because this is a wrapper, I'd rather do
```suggestion
        return await self.__wrapped__.__aexit__(exc_type, exc_val, exc_tb)
```